user-docs/latest/getting-started/introduction.md

```diff
@@ -22,14 +22,20 @@ The admin panel is the back office of your Strapi application. From the admin pa
 In order to access the admin panel, your Strapi application must be launched<!-- (see Installing Strapi) -->, and you must be aware of the URL to its admin panel (e.g. `api.example.com/admin`).
 :::
 
-![User profile](../assets/getting-started/login-page.png)
+![Login page with SSO activated](../assets/getting-started/login-page-SSO.png)
 
 To access the admin panel:
 
 1. Go to the URL of your Strapi application's admin panel.
 2. Enter your credentials to log in.
 3. Click on the **Log in** button. You should be redirected to the homepage of the admin panel.
 
+### Using SSO for authentication
```

Comment:
Hello @meganelacheny 
I read this page again and was wondering if we should indicate here that SSO is a Gold feature?

Reply:
Done!

Reply:
Thanks 🙂